src/client/pages/ZaakStatus/ZaakStatus.tsx

```diff
@@ -0,0 +1,121 @@
+import { useEffect, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { useAppStateGetter, useAppStateReady } from '../../hooks';
+import {
+  LinkdInline,
+  PageContent,
+  PageHeading,
+  TextPage,
+} from '../../components';
+import LoadingContent from '../../components/LoadingContent/LoadingContent';
+import { AppRoutes, DocumentTitles } from '../../../universal/config';
+import { isError, isLoading } from '../../../universal/helpers';
+
+const ITEM_NOT_FOUND = 'not-found';
+const STATE_ERROR = 'not-found';
```

Comment:
Dit is niet hetzelfde als not-found. Indien er een state error is weten we niet of het item bestaat of niet. Op alle andere thema pagina's tonen we dan een Alert.

```
<ErrorAlert>We kunnen op dit moment geen gegevens tonen.</ErrorAlert>
```